src/test/resources/cucumber/features/componentTests/webhook.feature

```diff
@@ -1,10 +1,9 @@
-@Skip @WebHookFeature @ComponentTest
+@WebHookFeature @ComponentTest
 Feature: Check Component tests Webhook functionality
 
   Scenario: High load on CxFlow due to frequent WebHook requests from GitHub
-    Given source is GitHub
-    And CxFlow is running as a service
+    Given version control provider is GitHub
+    Given CxFlow is running as a service
     And webhook is configured for push event
     When GitHub sends WebHook requests to CxFlow 2 times per second
-    Then 95% of the requests are answered in 200 ms or less
-    And the rest 5% of the requests are answered in between 200 ms and 500 ms
\ No newline at end of file
+    Then each of the requests is answered in at most 500 ms
```

Comment:
should we test benchmark tests with cucumber?
this way we ignore all the statistical fluctuations and other important aspects 👎

Reply:
This is basically a compromise between cucumber readability, code simplicity vs more advanced benchmark testing tools.
I suggest that we stay with cucumber for now.